src/Utilities/ToolLocationHelper.cs

```diff
@@ -61,6 +61,12 @@ public enum TargetDotNetFrameworkVersion
         Version451 = 6,
 
         /// <summary>
+        /// version 4.5.2. Enum is out of order because it was shipped out of band from a Visual Studio update
+        /// without a corresponding SDK release.
+        /// </summary>
+        Version452 = 9,
```

Comment:
:+1: for comment! I think I would prefer putting this in numeric order rather than logical, but I don't feel super strongly about it.